### PR TITLE
Update to the new WMF logo icon on About this app-page

### PR DESCRIPTION
**Pull request by vpeschenkov**
I hope my solution isn't awful :grimacing:

### CODE REVIEW

File: Wikipedia/about.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+function hideLightLogo() {
+    hide(document.getElementById("light-logo"))
+}
+
+function show(element) {
+    element.style.display = ''
+}
+
+function hide() {
+    element.style.display = 'none'
```
Comment (montehurd) [inline Wikipedia/about.css]: Missing `element` parameter here.

Comment (vpeschenkov) [reply Wikipedia/about.css]: Thanks. It's the wrong file (js code in .**css** file). Probably, it was added accidentally. I'll remove it.

**Comment by vpeschenkov** [comment]
@montehurd thanks for your answer. It's very useful! I've moved my changes into wikipedia-ios/www directory and tested it by the grunt. All works fine. Please, let me know if something is wrong. And, thanks for your help!

File: Wikipedia/Code/AboutViewController.m [hunk 2]
```diff
@@ -80,6 +80,18 @@ - (void)wmf_setTextFontColor:(WMFTheme *)theme {
     [self evaluateJavaScript:fontColorJS completionHandler:nil];
 }
 
+- (void)wmf_setLogoStyleWithTheme:(WMFTheme *)theme {
+    // White logo on Dark mode
+    // Black logo on Default and Sepia modes
+    if ([theme.name isEqualToString:WMFTheme.dark.name]) {
```
Comment (montehurd) [inline Wikipedia/Code/AboutViewController.m]: Oh I forgot you'll need to also use the white logo on darkDimmed mode:

> [theme.name isEqualToString:WMFTheme.darkDimmed.name]

Comment (montehurd) [reply Wikipedia/Code/AboutViewController.m]: screen shot 2017-09-22 at 11 24 36 am

screen shot 2017-09-22 at 11 24 43 am

Comment (vpeschenkov) [reply Wikipedia/Code/AboutViewController.m]: Thanks. I've added it.

**Pull request merged**